source/models/table.py

```diff
@@ -38,12 +38,11 @@ def get_name(self):
         return self.__table_name
 
     def set(self, row):
-        self.__colomns_name_validate(row = row)
         row_obj = Row(table = self, value = row)
         if row_obj.row_exists() and self.table_metadata.enable_overwrite == "false":
             raise RowExists(message = "this row is already exists and can't overwrite")
         elif row_obj.row_exists():
-            old_row = Row(table = self).load_by_primary_key(table=self, primary_key = row_obj.primary_key)
+            old_row = Row(table = self).load_by_primary_key(table = self, primary_key = row_obj.primary_key)
```

Comment:
Load is a static method, why do u need to create an object to call it?
It should be Row.load

Reply:
done